diff --git a/src/frontend/src/features/auth/utils/silentLogin.ts b/src/frontend/src/features/auth/utils/silentLogin.ts
index 3761a34..a4835cd 100644
--- a/src/frontend/src/features/auth/utils/silentLogin.ts
+++ b/src/frontend/src/features/auth/utils/silentLogin.ts
@@ -22,7 +22,7 @@ const initiateSilentLogin = () => {
 }
 
 export const canAttemptSilentLogin = () => {
-  return isRetryAllowed()
+  return false
 }
 
 export const attemptSilentLogin = (retryIntervalInSeconds: number) => {
